docs/install/install-from-packages.md

```diff
@@ -7,453 +8,8 @@ myst:
-make[1]: *** [Makefile:46: install-backend] Error 2
-make[1]: Leaving directory '/home/username/projects/volto/plone-volto'
-```
-
-You must delete your project, {ref}`follow the important note <avoid-plone-core-package-names>`, and run the cookiecutter again.
-````
-
-
-(install-packages-start-plone-label)=
-
-## Start Plone
-
-Plone 6 has two servers: one for the frontend, and one for the backend.
-As such, we need to maintain two active shell sessions, one for each server, to start your Plone site.
-
-
-(install-packages-start-plone-backend-label)=
-
-### Start Plone backend
-
-In the currently open session, issue the following command.
-
-```shell
-make start-backend
-```
-
-The Plone backend server starts up and emits messages to the console.
-
-```console
-2022-09-24 01:30:17,799 WARNING [ZODB.FileStorage:411][MainThread] Ignoring index for /<path-to-project>/my_project/project-title/backend/instance/var/filestorage/Data.fs
-2022-09-24 01:30:19,639 INFO    [chameleon.config:38][MainThread] directory cache: /<path-to-project>/my_project/project-title/backend/instance/var/cache.
-2022-09-24 01:30:23,680 INFO    [plone.volto:22][MainThread] Aliasing collective.folderish classes to plone.volto classes.
-2022-09-24 01:30:24,935 INFO    [Zope:42][MainThread] Ready to handle requests
-Starting server in PID 92714.
-2022-09-24 01:30:24,940 INFO    [waitress:486][MainThread] Serving on http://[::1]:8080
-2022-09-24 01:30:24,940 INFO    [waitress:486][MainThread] Serving on http://127.0.0.1:8080
-```
-
-
-(install-packages-start-plone-frontend-label)=
-
-### Start Plone frontend
-
-Create a second shell session in a new window.
-Change your current working directory to {file}`project-title`.
-Start the Plone frontend with the following command.
-
-```shell
-make start-frontend
-```
-
-The Plone frontend server starts up and emits messages to the console.
-
-```console
- WAIT  Compiling...
-
-
-✔ Client
-  Compiled successfully in 864.83ms
-
-✔ Server
-  Compiled successfully in 9.62s
-
-✅  Server-side HMR Enabled!
-sswp> Handling Hot Module Reloading
-Volto is running in SEAMLESS mode
-Using internal proxy: http://localhost:3000 -> http://localhost:8080/Plone
-🎭 Volto started at 0.0.0.0:3000 🚀
-```
-
-Note that the Plone frontend uses an internal proxy server to connect with the Plone backend.
-Open a browser at the following URL to visit your Plone site.
-
-http://localhost:3000
-
-You will see a page similar to the following.
-
-```{image} /_static/plone-home-page.png
-:alt: Plone home page
-:class: figure
-```
-
-Select the {guilabel}`Login` link to visit the login form, and enter the following credentials.
-
--   {guilabel}`Login name`: `admin`
--   {guilabel}`Password`: `admin`
-
-```{image} /_static/plone-login-page.png
-:alt: Plone login page
-:class: figure
-```
-
-Now you can edit content or configure your Plone site.
-
-You can stop the site with {kbd}`ctrl-c`.
 
-Enjoy!
+This page has been moved and renamed to {doc}`create-project`.
```

Comment:
It looks like https://documatt.com/sphinx-reredirects/index.html might be a nicer way to do this (so that it automatically redirects to the new location)

Reply:
(I would leave that for another PR though. Not a blocker.)

Reply:
Yeah, I was hesitating about that because there is an open PR https://github.com/plone/documentation/pull/1244. I think we can proceed, using that extension anyway.

I will also ask the @plone/ai-team to install a proper 301 redirect on the server after this is merged, as I do not have permission to do that. Once the 301 is installed, I can remove the original file.

Reply:
Done in a2c12c65b80ca77f578595bd23f6c60ef37e6c59

Reply:
Also asked @plone/ai-team in Discord.